twindb_backup/destination/s3.py

```diff
@@ -277,7 +279,7 @@ def _validate_upload(self, object_key):
 
         log.debug("Upload successfully validated")
 
-        return True
+        return 0
```

Comment:
innobackupex returns zero on success when we backup MySQL. When we backup files `ssh` also returns 0 on success.
So, we need consistent behavior from s3 as well. I don't mind returning True in all cases, but then we need to change it in other places.